src/csharp/Grpc.nuspec

```diff
@@ -3,18 +3,17 @@
   <metadata>
     <id>Grpc</id>
     <title>gRPC C#</title>
-    <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
     <version>$version$</version>
-    <authors>Google Inc.</authors>
-    <owners>grpc-packages</owners>
+    <authors>gRPC authors</authors>
```

Comment:
Can you create a separate PR for cleanup like this?

Reply:
With this particular file, it's a bit complicated. Examples depend on another change in it to compile properly, and this is just a coincidental one-line nit fix. But I get your overall point. split this into as many smaller and easier to review parts. I do not like its size either.

Reply:
And have fun on your vacation! I think I did not have one in 3 years or so... :-/